scripts/website/build-examples.sh

```diff
@@ -7,11 +7,26 @@ set -e # exit when error
 rm -rf website/examples
 mkdir website/examples
 
-for example in examples/*; do
-  if [ -d "$example" ]; then
-    name=$(basename "$example")
-    echo "Building $name example..."
+build_example() {
+  example=$1
+  name=$2
```

Comment:
I would have just moved `name=$(basename "$example")` in here in that case.
otherwise looks good 👍